ansi/buffer_test.go

```diff
@@ -15,7 +15,8 @@ func TestBuffer_PrintableRuneWidth(t *testing.T) {
 	}
 }
 
-func BenchmarkPrintableRuneWidth(b *testing.B) {
+// go test -bench=Benchmark_PrintableRuneWidth -benchmem -count=4
+func Benchmark_PrintableRuneWidth(b *testing.B) {
```

Comment:
Does the underscore here make any difference I'm not aware of? If not I'd suggest to keep the old name. I only used it to differentiate between type and method name, but that's not the case here. I guess the name of the test method above may have been a bit misleading, which also doesn't quite follow the style. Not a biggie one way or another, just curious.

Reply:
This is just personal taste, I prefer Benchmark[_struct]_function|method.

Reply:
No worries!